cpp/oneapi/dal/algo/svm/common.hpp

```diff
@@ -118,7 +118,7 @@ class descriptor_base : public base {
     double get_cache_size() const;
 
     /// The parameter of the WSS scheme $\\tau$.
-    /// @invariant :expr:`tau >= 0.0`
+    /// @invariant :expr:`tau > 0.0`
```

Comment:
Need to update message https://github.com/oneapi-src/oneDAL/blob/master/cpp/oneapi/dal/detail/error_messages.cpp#L156